bridge_adaptivity/module/engines/engine_vpal.py

```diff
@@ -143,4 +113,21 @@ def submit_activity_answer(self, sequence_item):
         submit_url = urlparse.urljoin(self.base_url, 'score')
         payload = self.fulfill_payload(instance_to_parse=sequence_item)
         submit_activity_score = requests.post(submit_url, json=payload, headers=self.headers)
-        return self.check_engine_response(submit_activity_score.status_code, 'graded', sequence_item.activity.name)
+        return self.check_engine_response(
+            submit_activity_score.status_code, action='graded', obj='sequence item', name=sequence_item.activity.name
```

Comment:
trailing comma?

Reply:
Not in this case. Within any changes in this line, it will be added to the commit diff view anyway.